pkgs/development/libraries/kde-frameworks/kdesu-search-for-wrapped-daemon-first.patch

```diff
@@ -0,0 +1,38 @@
+From 01af4d2a098e5819c09bca37568941dcd4b89d0b Mon Sep 17 00:00:00 2001
```

Comment:
Please create a directory for `kdesu` in `kde-frameworks` and move this patch there.

Reply:
Done.